test/integration/src/py/test_sifnode_peggy_pause_unpause.py

```diff
@@ -0,0 +1,114 @@
+    # TODO: sif_tx_fee vs get from envctx vs more lenient assertion
+    gas_cost = 160000000000 * 393000 # Taken from peggy1
+    assert balance_diff.get(sifchain.ROWAN, 0) == (-1 * sifchain.sif_tx_fee_in_rowan), "Gas fee should be deducted from sender sif acct"
+    assert balance_diff.get(sifchain.CETH, 0) == (-1 * (send_amount + gas_cost)), "Sent amount should be deducted from sender sif acct ceth balance"
+    assert erc_diff == send_amount, "Eth destination should receive rowan token"
+
+    res = ctx.sifnode.pause_peggy_bridge(ctx.sifchain_ethbridge_admin_account)
+    assert res[0]['code'] == 0
+
+    send_amount = 1
+    balance_diff, erc_diff = send_test_account(ctx, test_sif_account, test_eth_destination_account, send_amount, denom=sifchain.CETH, erc20_token_addr=None)
+    assert balance_diff.get(sifchain.ROWAN, 0) == (-1 * sifchain.sif_tx_fee_in_rowan), "Only gas fee should be deducted for attempted tx"
+    assert balance_diff.get(sifchain.CETH, 0) == 0, "Eth amount should'nt be deducted, no tx to evm"
+    assert erc_diff == 0, "Eth destination should not receive rowan token"
+
+
+    res = ctx.sifnode.unpause_peggy_bridge(ctx.sifchain_ethbridge_admin_account)
+    assert res[0]['code'] == 0
+
+    send_amount = 15
+    balance_diff, erc_diff = send_test_account(ctx, test_sif_account, test_eth_destination_account, send_amount, denom=sifchain.CETH, erc20_token_addr=None)
+    # TODO: sif_tx_fee vs get from envctx vs more lenient assertion
+    gas_cost = 160000000000 * 393000 # Taken from peggy1
+    assert balance_diff.get(sifchain.ROWAN, 0) == (-1 * sifchain.sif_tx_fee_in_rowan), "Gas fee should be deducted from sender sif acct"
+    assert balance_diff.get(sifchain.CETH, 0) == (-1 * (send_amount + gas_cost)), "Sent amount should be deducted from sender sif acct ceth balance"
+    assert erc_diff == send_amount, "Eth destination should receive rowan token"
+
+# TODO: Naming is terrible
+def send_test_account(ctx: EnvCtx, test_sif_account, test_eth_destination_account, send_amount, denom=sifchain.ROWAN, erc20_token_addr: str=None) -> Tuple[cosmos.Balance, int]:
+    sif_balance_before = ctx.get_sifchain_balance(test_sif_account)
+    if erc20_token_addr is not None:
+        eth_balance_before = ctx.get_erc20_token_balance(erc20_token_addr, test_eth_destination_account)
+    else:
+        eth_balance_before = ctx.eth.get_eth_balance(test_eth_destination_account)
+
+    ctx.sifnode_client.send_from_sifchain_to_ethereum(test_sif_account, test_eth_destination_account, send_amount, denom)
+
+    sif_balance_after = ctx.sifnode.wait_for_balance_change(test_sif_account, sif_balance_before)
+    try:
+        eth_balance_after = ctx.wait_for_eth_balance_change(test_eth_destination_account, eth_balance_before, token_addr=erc20_token_addr, timeout=30)
+    except Exception as e:
+        eth_balance_after = eth_balance_before
```

Comment:
Should we not fetch the user balance rather than assuming it has not changed if an exception is raised?

Reply:
`wait_for_eth_balance_change` will always return value != old_balance, or it will raise Exception. When we receive an exception we know we've timed out, and new_balance == old_balance

One improvement is to assert it is a Timedout exception. Jure and I are working on that 

This function will be moved into siftool as well as part of send. It wont return until balance is changed